packages/tenant-process/src/services/tenantService.ts

```diff
@@ -699,4 +719,54 @@ function reassignDeclaredAttribute(
   );
 }
 
+function assignVerifiedAttribute(
+  targetTenant: Tenant,
+  organizationId: TenantId,
+  tenantAttributeSeed: ApiVerifiedTenantAttributeSeed
+): TenantAttribute[] {
+  return [
+    ...targetTenant.attributes,
+    {
+      id: unsafeBrandId(tenantAttributeSeed.id),
+      type: tenantAttributeType.VERIFIED,
+      assignmentTimestamp: new Date(),
+      verifiedBy: [
+        {
+          id: organizationId,
+          verificationDate: new Date(),
+          expirationDate: tenantAttributeSeed.expirationDate
+            ? new Date(tenantAttributeSeed.expirationDate)
+            : undefined,
+          extensionDate: tenantAttributeSeed.expirationDate
+            ? new Date(tenantAttributeSeed.expirationDate)
+            : undefined,
+        },
+      ],
+      revokedBy: [],
+    },
+  ];
+}
+
+function reassignVerifiedAttribute(
+  targetTenant: Tenant,
+  verifiedTenantAttribute: VerifiedTenantAttribute,
+  organizationId: TenantId,
+  tenantAttributeSeed: ApiVerifiedTenantAttributeSeed
+): TenantAttribute[] {
+  return targetTenant.attributes.map((attr) =>
+    attr.id === verifiedTenantAttribute.id
+      ? {
+          ...attr,
+          assignmentTimestamp: verifiedTenantAttribute.assignmentTimestamp,
+          verifiedBy: buildVerifiedBy(
+            verifiedTenantAttribute,
+            organizationId,
+            tenantAttributeSeed
+          ),
+          revokedBy: verifiedTenantAttribute.revokedBy,
```

Comment:
```suggestion
          verifiedBy: buildVerifiedBy(
            verifiedTenantAttribute,
            organizationId,
            tenantAttributeSeed
          ),
```
Since `attr` and `verifiedTenantAttribute` are the same attribute here, you are already adding these fields with `...attr`